library/core/game_test/src/main/java/org/quiltmc/qsl/game_test/mixin/server/MainMixin.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright 2016, 2017, 2018, 2019 FabricMC
+ * Copyright 2022 QuiltMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.quiltmc.qsl.game_test.mixin.server;
+
+import java.io.File;
+import java.nio.file.Path;
+
+import joptsimple.OptionParser;
+import joptsimple.OptionSet;
+import joptsimple.OptionSpec;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.Redirect;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+import org.spongepowered.asm.mixin.injection.callback.LocalCapture;
+
+import net.minecraft.resource.pack.ResourcePackManager;
+import net.minecraft.server.Main;
+import net.minecraft.server.Services;
+import net.minecraft.server.dedicated.EulaReader;
+import net.minecraft.server.dedicated.ServerPropertiesLoader;
+import net.minecraft.world.storage.WorldSaveStorage;
+import net.minecraft.world.storage.WorldSaveSummary;
+
+import org.quiltmc.loader.api.minecraft.DedicatedServerOnly;
+import org.quiltmc.qsl.game_test.impl.QuiltGameTestImpl;
+
+@DedicatedServerOnly
+@Mixin(Main.class)
+public class MainMixin {
+	@Redirect(method = "main", at = @At(value = "INVOKE", target = "Lnet/minecraft/server/dedicated/EulaReader;isEulaAgreedTo()Z"))
+	private static boolean isEulaAgreedTo(EulaReader reader) {
+		return QuiltGameTestImpl.ENABLED || reader.isEulaAgreedTo();
```

Comment:
Do we need/have some kind of "By running these tests you agree to the Minecraft EULA" message if we're doing this?